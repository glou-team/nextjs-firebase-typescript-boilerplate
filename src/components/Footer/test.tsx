import { render, screen } from '@testing-library/react'

import Footer from '.'

describe('<Footer />', () => {
  it('Should render the footer', () => {
    const { container } = render(<Footer />)

    expect(screen.getByText(/Powered by/i)).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
