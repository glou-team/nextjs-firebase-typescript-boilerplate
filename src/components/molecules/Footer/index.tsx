import * as S from './styles'

const Footer = ({ message = '' }) => {
  return <S.Container>{message}</S.Container>
}

export default Footer
