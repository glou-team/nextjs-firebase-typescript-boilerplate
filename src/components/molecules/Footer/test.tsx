import { render } from '@testing-library/react'

import Footer from '.'

describe('<Footer />', () => {
  it('Should render the footer', () => {
    const { container } = render(<Footer />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
