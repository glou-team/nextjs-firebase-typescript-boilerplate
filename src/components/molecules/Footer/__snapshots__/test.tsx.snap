// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Should render the footer 1`] = `
.c0 {
  width: 100%;
  height: 100px;
  border-top: 1px solid #eaeaea;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<footer
  class="c0"
/>
`
